.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Can you make sure that it's okay to add comments here? Asking since sometimes, you just cannot add comments in JSON files.

Reply:
yeah this comment is auto generated and handled by vscode :+1: